### PR TITLE
Alert Tab UI Tweaks

### PR DESCRIPTION
**Pull request by kingthorin**
AlertViewPanel.java > Disable horizontal scrolling, enable line wrap for text areas.

### CODE REVIEW

**Comment by kingthorin** [comment]
@psiinon look good?

**Comment by psiinon** [comment]
yep :)

File: src/org/zaproxy/zap/extension/alert/AlertViewPanel.java [hunk 1]
```diff
@@ -155,6 +156,7 @@ private void initialize() {
 	private JScrollPane getAlertPane() {
 		if (alertPane == null) {
 			alertPane = new JScrollPane();
+			alertPane.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/alert/AlertViewPanel.java]: This is an improvement but there's still an issue, the text areas are allowed to expand but they don't shrink, so if you resize the panel "too much" you might no longer see some of the data after restoring it to the previous size. I guess each text area needs its own scroll pane(?).

File: src/org/zaproxy/zap/extension/alert/AlertViewPanel.java [hunk 2]
```diff
@@ -274,6 +276,7 @@ public void actionPerformed(ActionEvent e) {
 			alertUrl.setLineWrap(true);
 			
 			alertDescription = createZapTextArea();
+			alertDescription.setLineWrap(true);
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/alert/AlertViewPanel.java]: It seems this is already being done in `createZapTextArea`.

**Comment by kingthorin** [comment]
Will look into further changes

**Comment by kingthorin** [comment]
This was discussed a bit today via IRC. For the record: https://mozilla.logbot.info/websectools/20180222

**Pull request merged**